Bin/Snapshot.php

```diff
@@ -264,6 +303,10 @@ function () use ($tags, $newTag, $repositoryRoot, &$changelog) {
                 } else {
                     array_unshift($tags, 'HEAD');
 
+                    $changelogThemesRegex = sprintf(
+                        '/(?<type>%s)\((?<section>[^)]+)\)(?<message>.*)$/',
```

Comment:
I would add the `^` anchor too.

Reply:
Since the commit line starts with some space and `*` characters, I've updated the regex with the `^` anchor and a `[ *]+` matcher.

Reply:
Does it happen? If yes, this is an issue. We must not have leading space to the commit title.

Reply:
It's about the `git log` command output. Here a sample output :

```
  * chore(composer) Use `dev-master` dependencies. (Ivan Enderlin, 2018-01-23T10:42:02+01:00)
```

Every line starts with these characters...

Reply:
Then check https://github.com/hoaproject/Devtools/pull/49/files#r165924646.